src/pages/lib/auth/manageusers/q/platform/[platform].mdx

```diff
@@ -1,11 +1,456 @@
 export const meta = {
-  title: `Password & user management`,
-  description: `Learn more about how to handle common password and user management scenarios. E.g. password reset, account recovery etc.`,
+  title: `Manage user profiles`,
+  description: `Learn more about how to enable customers to personalize their profile and verify their contact information with attributes.`,
 };
 
-import manageUsers from "/src/fragments/lib/auth/js/manageusers.mdx";
+<InlineFilter filters={["js", "react-native"]}>
 
-<Fragments fragments={{
-  js: manageUsers,
-  'react-native': manageUsers
-}} />
\ No newline at end of file
+User profile management helps you keep your applications secure while also personalizing your app experiences. In this guide we will review how you can enable your users to personalize their profile and verify their contact information. This includes outlining how you can set up user attributes, verify them, and allow your users to delete them when necessary.
+
+Before you begin, you will need:
+- An Amplify project with the Auth category configured
+- The Amplify libraries installed and configured
+
+## Set up user attributes
+
+Attributes such as name, email address, phone number help you identify individual users. Defining the user attributes you include for your user profiles makes user data easy to manage at scale. This information will help you personalize user journeys, tailor content, provide intuitive account control, and more. You can capture information upfront during sign-up or in a customizable profile. In this section we take a closer look at working with user attributes and how to set them up and manage them.
```

Comment:
you can capture infromation upfront during sign-up or "enable customers to update their profile after sign up"

what is in quotes is an update. Something along those lines. Customizable profile doesn't say that clearly to me.